src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -5,3 +5,22 @@
 #include "udfs/citus_finish_pg_upgrade/10.0-1.sql"
 
 #include "../../columnar/sql/columnar--9.5-1--10.0-1.sql"
+
+CREATE VIEW public.citus_tables AS
+SELECT
+  logicalrelid AS table_name,
+  CASE WHEN partkey IS NOT NULL THEN 'distributed' WHEN repmodel = 't' THEN 'reference' ELSE 'local' END AS citus_table_type,
```

Comment:
this `local` kind of exposes the citus local tables to users I guess, would that be confusing?

Reply:
Good point, we can leave them out.